php/class-plugin.php

```diff
@@ -190,7 +190,7 @@ private function get_settings_page_structure() {
 			'version'    => $this->version,
 			'page_title' => __( 'Cloudinary', 'cloudinary' ),
 			'menu_title' => __( 'Cloudinary', 'cloudinary' ),
-			'capability' => 'manage_options',
+			'capability' => Utils::user_can( 'manage_settings' ),
```

Comment:
Same as with the media library, will result in only admins using.
suggest:
```suggestion
			'capability' => Utils::user_can( 'manage_settings' )  ? 'exist' : false,
```